parts/k8s/kubernetesinstalls.sh

```diff
@@ -129,6 +129,27 @@ installDockerEngine() {
     fi
 }
 
+configureDockerEngine() {
```

Comment:
I would move this to kubernetesconfigs. All runtime configurations should be in https://github.com/Azure/aks-engine/blob/1bfacd8f278143989e938884b2af560203457a79/parts/k8s/kubernetesconfigs.sh as opposed to kubernetesinstalls since this doesn't involve internet outbound

Reply:
Tried to keep install and config for docker in same file, easier to search, but understand that its better off in kubernetesconfig.sh to keep in sync with code structure.

Reply:
i see that packer/install-dependencies only include the below files


source /home/packer/provision_installs.sh
source /home/packer/provision_source.sh

Doesnt seem to include kubernetesconfigs.sh

```
 {
            "type": "file",
            "source": "packer/cleanup-vhd.sh",
            "destination": "/home/packer/cleanup-vhd.sh"
        },
        {
            "type": "file",
            "source": "parts/k8s/kubernetesinstalls.sh",
            "destination": "/home/packer/provision_installs.sh"
        },
        {
            "type": "file",
            "source": "parts/k8s/kubernetesprovisionsource.sh",
            "destination": "/home/packer/provision_source.sh"
        },
        {
            "type": "file",
            "source": "packer/install-dependencies.sh",
            "destination": "/home/packer/install-dependencies.sh"
        },
```